site/content/en/docs/manual/advanced/analytics-and-monitoring/auto-qa.md

```diff
@@ -78,90 +164,179 @@ To create a **Ground truth** job, do the following:
      It can be any integer number, the same value will yield the same random selection (given that the
      frame number is unchanged). <br> **Note** that if you want to use a
      custom frame sequence, you can do this using the server API instead,
-     see [Jobs API #create](https://docs.cvat.ai/docs/api_sdk/sdk/reference/apis/jobs-api/#create).
+     see [Job API create()](https://docs.cvat.ai/docs/api_sdk/sdk/reference/apis/jobs-api/#create).
 
 4. Click **Submit**.
-5. Annotate frames, save your work.
-6. Change the status of the job to **Completed**.
-7. Change **Stage** to **Accepted**.
 
 The **Ground truth** job will appear in the jobs list.
 
-![Add new job](/images/honeypot03.jpg)
+![Ground Truth job](/images/honeypot03.jpg)
 
-## Managing Ground Truth jobs: Import, Export, and Deletion
+5. Annotate frames and save your work or upload annotations.
+6. Change the **Stage** of the job to **Acceptance**.
+7. Change the **Status** of the job to **Completed**.
 
-Annotations from **Ground truth** jobs are not included in the dataset export,
-they also cannot be imported during task annotations import
-or with automatic annotation for the task.
+![Set job status](/images/honeypot10.jpg)
 
-Import, export, and delete options are available from the
-job's menu.
+## Validation modes
 
-![Add new job](/images/honeypot04.jpg)
+Currently, there are 2 validation modes available for tasks: **Ground Truth** and **Honeypots**.
+These names are often used interchangeably, but in CVAT they have some differences.
+Both modes rely on the use of Ground Truth annotations in a task,
+stored in a [Ground Truth job](#ground-truth-jobs), where they can be managed.
 
-### Import
+### Ground Truth
 
-If you want to import annotations into the **Ground truth** job, do the following.
+In this mode some of the task frames are selected into the validation set, represented as a
+separate Ground Truth job. The regular annotation jobs in the task are not affected in any way.
 
-1. Open the task, and find the **Ground truth** job in the jobs list.
-2. Click on three dots to open the menu.
-3. From the menu, select **Import annotations**.
-4. Select import format, and select file.
-5. Click **OK**.
+Ground Truth jobs can be created and removed at the task creation automatically or
```

Comment:
What do you mean by the automatic removal of GT tasks? Is there any difference with GT jobs created in tasks with `gt_pool` validation mode?

Reply:
Fixed